### DISCUSSION TITLE
Specify postgres schema during installation?

### DISCUSSION THREAD

**Discussion by magick93** [medusajs/medusa#5465] [Q&A]
Hello

Is there anyway to specify a specific postgres schema when installing medusa?

Currently it defaults to `public` - which is understandable. However, I'm wanting to use it with an existing database, and there a clash of table names.

**Comment by rbkdigital** [comment]
I am having the same issue. Did you figure this out @magick93 ?

**Comment by aerskine-resonance** [comment]
not fixed in v2 as it seems like you can specifcy `databaseSchema` in medusa-config.ts but at the moment it only sets up the links to the specified schema and all other tables are still in public.
